tsconfig.json

```diff
@@ -22,18 +22,17 @@
     "removeComments": true,                   /* Do not emit comments to output. */
     // "noEmit": true,                        /* Do not emit outputs. */
     // "importHelpers": true,                 /* Import emit helpers from 'tslib'. */
-     "downlevelIteration": true,            /* Provide full support for iterables in 'for-of', spread, and destructuring when targeting 'ES5' or 'ES3'. */
+    "downlevelIteration": true,            /* Provide full support for iterables in 'for-of', spread, and destructuring when targeting 'ES5' or 'ES3'. */
     // "isolatedModules": true,               /* Transpile each file as a separate module (similar to 'ts.transpileModule'). */
 
     /* Strict Type-Checking Options */
     "strict": true,                           /* Enable all strict type-checking options. */
```

Comment:
I think it seems that strict was alreay on, is null check reinforcement added?

Reply:
The `strict` has been on, but its sub-option (`strictNullChecks`) has been off.
So, in the PR, I commented out all sub-options, so that the `strict` affects globally.
https://www.typescriptlang.org/tsconfig#strict
> Turning this on is equivalent to enabling all of the strict mode family options, which are outlined below

Reply:
OK. I got it. 😃